source/speechViewer.py

```diff
@@ -24,8 +27,26 @@ def __init__(self, onDestroyCallBack):
 		self.Bind(wx.EVT_CLOSE, self.onClose)
 		self.Bind(wx.EVT_WINDOW_DESTROY, self.onDestroy)
 		sizer = wx.BoxSizer(wx.VERTICAL)
-		self.textCtrl = wx.TextCtrl(self, -1,style=wx.TE_RICH2|wx.TE_READONLY|wx.TE_MULTILINE)
+
+		self.lineCount = 0
+		self.textCtrl = wx.TextCtrl(self, -1, style=wx.TE_RICH2|wx.TE_READONLY|wx.TE_MULTILINE)
+		fontPointSize = config.conf["speechViewer"]["fontPointSize"]
+		fontFamily = config.conf["speechViewer"]["fontFamily"]
+		fontStyle = config.conf["speechViewer"]["fontStyle"]
+		fontWeight = config.conf["speechViewer"]["fontWeight"]
+		fontFaceName = config.conf["speechViewer"]["fontFaceName"]
+		try:
+			font = wx.Font(fontPointSize, fontFamily, fontStyle, fontWeight, False, fontFaceName)
+			self.textCtrl.SetFont(font)
+		except:
+			log.error("Error while setting the speech viewer font, continuing on...", exc_info=True)

```

Comment:
I'm worried there's always an error like this as long as the font isn't explicitly changed. See the following example:

```
>>> font = wx.Font(None, None, None, None, False, None)
Traceback (most recent call last):
  File "<console>", line 1, in <module>
TypeError: Font(): arguments did not match any overloaded call:
  overload 1: too many arguments
  overload 2: argument 1 has unexpected type 'NoneType'
  overload 3: argument 1 has unexpected type 'NoneType'
  overload 4: argument 1 has unexpected type 'NoneType'
  overload 5: argument 1 has unexpected type 'NoneType'
  overload 6: argument 1 has unexpected type 'NoneType'
  overload 7: argument 1 has unexpected type 'NoneType'
```

I think catching an error like this is ok as it is still logged, but we need to ensure that this code does not usually fail.